runner.go

```diff
@@ -22,11 +25,21 @@ import (
 const SplunkUser = "admin"
 const SplunkHost = "127.0.0.1:8089"
 const SplunkPath = "${SPLUNK_HOME}/bin/splunk"
-const SplunkCACert = "${SPLUNK_HOME}/etc/auth/cacert.pem"
+const SplunkAppsCACert = "${SPLUNK_HOME}/etc/auth/appsCA.pem"
+const SplunkCACert = "${SPLUNK_HOME}/etc/auth/ca.pem"
```

Comment:
I assume the change to file path is in a related change in the operator?

Reply:
Yes